src/components/WebexMeeting/WebexMeeting.stories.js

```diff
@@ -1,26 +1,29 @@
 import React from 'react';
-import {storiesOf} from '@storybook/react';
+import WebexMeeting from './WebexMeeting';
 
-import jsonData from '../../data';
-import {WebexJSONAdapter} from '../../adapters';
+export default {
+  title: 'Meetings/Webex Meeting',
+  component: WebexMeeting,
+};
 
-import {WebexMeeting, WebexDataProvider} from '..';
+const Template = (args) => <WebexMeeting {...args} />;
```

Comment:
I've found that the current implementation of the WebexMeeting and child components currently rely on a container of a set size. The min height css does not propagate down and causes it to look like this:

![image](https://user-images.githubusercontent.com/190647/93618398-31e17280-f9a5-11ea-95ca-4bb781bf5471.png)

Adding the fixed width container gets us a more working example:

```suggestion
const Template = (args) => <div style={{width: '650px', height: '500px'}}><WebexMeeting {...args} /></div>;
```

Reply:
I'm aware of this issue. I also found some other styling issues that I wanted to fix but I wanted to open a separate PR for it since this was too unwieldy already

Reply:
Since the code was already in for creating a fixed size container, let's continue using that strategy instead of removing it and breaking the display.

Reply:
Added the styles back in the mean time